### ISSUE TITLE
URGENT! Geoengine, useless without this fix

### ISSUE THREAD

**Issue by Pavel2451** [L2J/L2J_Server#154]
Hello, we need urgent fix for geoengine !
Please fix it. Server files are useless... big problem.

**Comment by Sacrifice78** [comment]
We're waiting yet, yes.

**Comment by Pavel2451** [comment]
Any progress with Geoengine?

**Comment by luksdlt92** [comment]
For now, use the last stable version.

**Comment by Pavel2451** [comment]
Where I can find it ?

**Comment by luksdlt92** [comment]
Check the releases https://github.com/L2J/L2J_Server/releases

**Comment by Pavel2451** [comment]
Its seems same...

**Comment by Pavel2451** [comment]
Tell me what version I can use ?

**Comment by luksdlt92** [comment]
I'm not sure, i don't know where the bug was introduced.

**Comment by Zoey76** [comment]
This report doesn't say what is the issue...

**Comment by Pavel2451** [comment]
@Zoey76  hi, first topic updated

**Comment by impos89** [comment]
Well as I remember the bug with bee hive is really old. I was fixed it for private project probably in 2010... It is needed to fix manually the geodata cell. Unfortunately I have no access to this fix anymore.

**Comment by Pavel2451** [comment]
yes i know, i was the fixed too, but the problem is with geoengine. Need to check if working correctly.

**Comment by Zoey76** [comment]
Christian will be checking this and other geodata and pathfinding issues.

**Comment by Sacrifice78** [comment]
:+1:

**Comment by Pavel2451** [comment]
Thanks you :)

**Comment by maneco2** [comment]
Only to remember @Zoey76 , Christian should use V2 geodata to see problems :+1:

**Comment by Sacrifice78** [comment]
Yes!. Totally true.

**Comment by HorridoJoho** [comment]
@maneco2 I totally planned to do this. The geo engine is supposed to work perfectly with the V2 geodata.

**Comment by Pavel2451** [comment]
where i can download v2 geo?

**Comment by Sacrifice78** [comment]
Absolutely incredible!!! @HorridoJoho... +1

**Comment by Pavel2451** [comment]
When will be finished app +- time?

**Comment by HorridoJoho** [comment]
@Pavel2451 I'm afraid i can't give you an estimated time of arival. When everything went's well i have analized the problems by the end of the coming week and then i can start to fix things up.

**Comment by maneco2** [comment]
Yes @HorridoJoho, we all know that V2 Geodata works perfectly :+1: , it's only matter of fixing geo engine

**Comment by Pavel2451** [comment]
Thanks you. What is your recommend to me wht to use?  What Geo is working with current geoegine ?

**Comment by HorridoJoho** [comment]
@Pavel2451 I can not answere you that question. Maybe another team member can give you an answere or even the community. @Zealar @Zoey76

**Comment by maneco2** [comment]
recomended geodata?
